tests/snappi/pfc/files/helper.py

```diff
@@ -428,18 +427,18 @@ def __verify_results(rows,
         speed_gbps (int): link speed in Gbps
         test_flow_pause (bool): if test flows are expected to be paused
         tolerance (float): maximum allowable deviation
+        pfc_pause_zero_src_mac (bool): if pause frames have zero source MAC
```

Comment:
It seems that "pfc_pause_zero_src_mac" argument is not needed. No matter if PFC pause frames have zero source MAC, we should check if PFC counters are correct.